Dockerfile

```diff
@@ -1,10 +1,9 @@
 
 # --platform=$BUILDPLATFORM is used build javascript source with host arch
 # Otherwise TS builds on emulated archs and can be extremely slow (+1h)
-FROM --platform=${BUILDPLATFORM:-amd64} node:20-alpine as build_src
+FROM --platform=${BUILDPLATFORM:-amd64} node:20 as build_src
```

Comment:
why are these images changed? Iirc they're intentionally alpine for the smaller image size.

Reply:
Those are build stage images, and do not end up in the final generated image. The one we care about is the last one, which is kept as an alpine.
Moving to non-alpine images reduces the need to install extra tooling via `apk`, which cannot be cached and significantly slow down the build process.